src/scss/_osu_layout_builder.scss

```diff
@@ -12,3 +12,8 @@
 #drupal-modal .form-item {
   width: initial !important;
 }
+
+.input-icon.fade-zoom-in {
+  background-image: url(themes/custom/madrone/assets/fade-zoom-in.svg);
```

Comment:
Make path relative to where the final CSS file will live
background-image: url("../assets/fade-zoom-in.svg");

makes it more robust to installation source

Reply:
Thanks. I had a question on, it wasn't clear what the path there should be.